### DISCUSSION TITLE
Time series with CSV data

### DISCUSSION THREAD

**Discussion by rajdevworks** [grafana/grafana-infinity-datasource#897] [General]
I have date column in format 2023/09/26 10:19:27 and I tried doing a bar chart to get a count of entries to see by day/month. It is not working for me. What is best way to visualize a time series based on date/time column and perform count per day/month for column X/Y/Z

Reference slack thread - https://grafana.slack.com/archives/C0Y4TLW74/p1719407263000639

**Comment by yesoreyeram** [comment]
Can you also post a sample data?

**Comment by rajdevworks** [comment]
[test-csv.csv](https://github.com/user-attachments/files/15995301/test-csv.csv)

**Comment by rajdevworks** [comment]
@yesoreyeram can this be visualized with the dataset I have or do I need to reformat/transform my data?
